api/src/main/java/org/apache/iceberg/exceptions/AlreadyExistsException.java

```diff
@@ -25,6 +25,10 @@
  * Exception raised when attempting to create a table that already exists.
  */
 public class AlreadyExistsException extends RuntimeException {
+  public AlreadyExistsException(String message) {
+    super(message);
```

Comment:
Is this needed? I think `args` in the next constructor can simply be empty, right?